test/Interop/WebAssembly/extern-wasm-cdecls.swift

```diff
@@ -0,0 +1,36 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend %s -emit-ir -module-name Extern | %FileCheck %s
+
+// CHECK: declare void @"$s6Extern7import1yyF"() [[EA1:#[0-9]+]]
+@_extern(c)
+@_extern(wasm, module: "m0", name: "import1")
+func import1()
+
+// CHECK: declare i32 @"$s6Extern20import2WithReturnInts5Int32VyF"() [[EA2:#[0-9]+]]
+@_extern(c)
+@_extern(wasm, module: "m0", name: "import2")
+func import2WithReturnInt() -> Int32
+
+// CHECK: declare void @"$s6Extern16import3TakingIntyys5Int32VF"(i32) [[EA3:#[0-9]+]]
+@_extern(c)
+@_extern(wasm, module: "m0", name: "import3")
+func import3TakingInt(_: Int32)
```

Comment:
These look like they'll fail as a result of the change of naming (sorry).

Reply:
Oops, I didn't checked Interop tests 😅